Reading List/Reading List/View Controllers/BookDetailViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  BookDetailViewController.swift
+//  Reading List
+//
+//  Created by Norlan Tibanear on 6/6/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookDetailViewController: UIViewController {
+    
+    // Outlets
+    @IBOutlet var titleTextField: UITextField!
+    @IBOutlet var descriptionTextView: UITextView!
+    
+    var bookController: BookController?
+    var book: Book?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        descriptionTextView.layer.borderWidth = 0.5
+        descriptionTextView.layer.borderColor = UIColor.lightGray.cgColor
+        descriptionTextView.clipsToBounds = true
```

Comment:
I love that you are always tweaking the UI a bit to your liking. A radius on the corners as well would have made it more cohesive with the text field (subjectively speaking).